sacred/config/custom_containers.py

```diff
@@ -185,10 +187,8 @@ class ReadOnlyDict(ReadOnlyContainer, dict):
     __setitem__ = ReadOnlyContainer._readonly
     __delitem__ = ReadOnlyContainer._readonly
 
-    def __init__(self, *args, message=None, **kwargs):
-        if message is None:
-            message = "This ReadOnlyDict is read-only!"
-        super().__init__(*args, message=message, **kwargs)
+    def __init__(self, d):
+        dict.__init__(self, d)
```

Comment:
The `__init__` can now be removed. As long as `ReadOnlycontainer` doesn't have an `__init__` method, it should work without explicitly passing it on to `dict`. Then, we naturally keep the signature of `dict.__init__`

Reply:
And the same for the list, but the comment somehow didn't make it here

Reply:
Ah yes, thanks